seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/SeaTunnelTask.java

```diff
@@ -152,12 +151,6 @@ protected void checkDone() {
         }
     }
 
-    @Override
-    public void close() throws IOException {
-        startFlowLifeCycle.close();
-        progress.done();
-    }
-
```

Comment:
Does `TransformSeaTunnelTask` not need to override this method?

Reply:
Right, it close by `checkClose()`